public/configs/v1/env.json

```diff
@@ -157,7 +159,8 @@
          "protocolStaking": "[HTTP link to protocol staking info]",
          "stakingAndClaimingRewardsLearnMore": "[HTTP link to staking and claiming rewards learn more]",
          "predictionMarketLearnMore": "[HTTP link to prediction market learn more]",
-         "discoveryProgram": "[HTTP link to discovery program learn more]"
+         "discoveryProgram": "[HTTP link to discovery program learn more]",
+			"getInTouch": "[HTTP link to get in touch with traders]"
```

Comment:
is the spacing still off 👀

Reply:
okkk i will follow up on this HAHA for the sake of not re-requesting review and blocking the notif